docs/group_concepts/rg_scoping.rst

```diff
@@ -0,0 +1,23 @@
+Azure Resource Group Scoped Private Datacenter 
+========================================
+
+In PubSub+ Cloud, you need to determine which datacenter will be used to manage your new messaging service broker. These datacenters are either public or private and located on different clouds, ie AWS, GCZ, or Azure.  Private datacenters cannot be used by any other organizations whereas pubic datacenter can be shared between many organzations.  
+For Public Azure Cloud, any time you create a new message broker, Solace will first allocate a new resource group and then deploy the message broker's infrastrcuture within the resource group. Deletion of resource group occurs on deletion of the message broker.
+If you are a Enterprise customer with applications in Azure Cloud, you may want a private Azure datacenter in your own tenat. This allows you to set your own policies and security protocols.  For further security, you may be interested in a resource group scoped private data center.
+
+To understand the benefit of resource group scoped private datacenters it is important to understand some high-level concepts about how a private datacenter operates:
+
+A private datacenter lives inside your tenant. It requires certain permissions to Azure resources so that it can manage message brokers on your behalf. This neccesitates the creation of an application registration with proper roles assigned to give the app access to various Azure resources such as VMs, subnets, etc. The datacenter then uses this application registration to perform deployments neccesary for creation, update, deletion of brokers.
```

Comment:
```suggestion
A private datacenter lives inside your subscription. To successfully orchestrate service, It requires certain permissions to Azure resources so that it can manage message brokers on your behalf. This necessitates the creation of an application registration with proper roles assigned to give the app access to various Azure resources such as VMs, subnets, etc. The datacenter then uses this application registration to perform deployments necessary for creation, update, deletion of brokers.
```